scripts/terraform-docs.awk

```diff
@@ -0,0 +1,90 @@
+# This script converts Terraform 0.12 variables/outputs to something suitable for `terraform-docs`
```

Comment:
Instead of copying these scripts into each project, let's pull them from the central project we setup to store them:

https://github.com/plus3it/terraform-aws-tardigrade-iam-principals/blob/master/Makefile#L103-L106